news/_posts/2022-02-14-new-theme-announce.md

```diff
@@ -58,18 +56,20 @@ Have a look at the [mockups](https://www.figma.com/file/LfVl5leeSKcVUhSaYwhbtM/N
 
 As this new theme is getting bigger and more complete, we must consider what will happen for the Classic Theme when this new theme is ready.
 
-We do not plan to maintain both themes simultaneously, so when the new theme rises, it will be necessary to decide how to stop the support of the Classic Theme. Today we have not yet decided how this will happen.
+The classic theme has a [dedicated repository](https://github.com/PrestaShop/classic-theme) and should be compatible with 1.7 and 8 versions.
+
+It is not planned to maintain both themes simultaneously, so once the new theme has graduated to become the default theme for new releases, the support of the Classic Theme will be over.
 
-Rest assured we will make sure that the end-of-life of Classic theme and the transition to this new theme will be made very smooth.
+Rest assured we will make sure that the end-of-life of Classic theme and the transition to this new theme will be made in the smoothest possible way.
 
 ## Your feedback is a gift
 
-You can vote to choose the name of this new theme [here](https://framaforms.org/vote-for-the-new-theme-name-1644576025).
+You can vote until February 18, 2022, to choose the name of this new theme [here](https://framaforms.org/vote-for-the-new-theme-name-1644576025).
```

Comment:
I suggest we give it one more week and rephrase. You should never use "this" or "here" for links because it's bad for SEO and screen readers.
```suggestion
You can [vote to choose the name of this new theme](https://framaforms.org/vote-for-the-new-theme-name-1644576025) until February 23, 2022.
```